### PR TITLE
QRadar enrich closing reason with note of a mirrored incident

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39610

## Description
Added more enriched closing message for closed incident mirrored by qradar offense

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
When someone closes an offense from QRadar, it can be from QRadar UI or an API call. When closing from QRadar UI, it asks you to add a note for the close. 
This change takes this into consideration as customer said he would like to see the note his analysts give him for the closed offense because closing reason itself is not sufficient for him.

Adding Screenshots:

**Before**:
- Screenshots of an offense being closed before this PR change:
Screen Shot 2021-07-28 at 16 20 07


**After change**:
- Screenshots of an offense being closed using QRadar UI + leaving note now:(reason = Policy Violation):
Screen Shot 2021-07-28 at 16 08 03

- Screenshots of an offense being closed using QRadar UI without a note:(reason = False Positive Tuned)
Screen Shot 2021-07-28 at 16 10 04

- Screenshots of an offense being closed using QRadar API:(remains the same as before the change)
Screen Shot 2021-07-28 at 16 08 34

**Comment by tomneeman151293** [comment]
@DinaMeylakh FYI

**Comment by tomneeman151293** [comment]
Ignore gitlab failures, it is the connectivity issues. this is ready for CR

File: Packs/QRadar/ReleaseNotes/2_0_18.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### IBM QRadar v3
+- Updated the Docker image to: *demisto/python3:3.9.6.22912*.
+- Enhanced the closing reason message of a mirrored incident when being closed.
```
Comment (roysagi) [inline Packs/QRadar/ReleaseNotes/2_0_18.md:5]: Seems kind of vague - lets try and make it clearer since it is mirroring related with some guideline questions:
1 - Which side is closing and which is being closed?
2 - Is this relevant to all mirroring incidents or just some?
3 - I think "closing reason" is an incident field no? if so it should be written like in the system and covered with **on each side**.

Comment (tomneeman151293) [reply Packs/QRadar/ReleaseNotes/2_0_18.md:5]: 1) in README it says, QRadar supports only mirror in (QRadar to XSOAR), so I thought it was clear.
2) All mirroring incidents
3) I talked about the XSOAR incident closing reason (as it is being printed when closing investigation).

Comment (roysagi) [reply Packs/QRadar/ReleaseNotes/2_0_18.md:5]: ok cool so lets just be explicit about this:
```
- Improved the outputted closing reason message when a mirrored incident is being closed (from QRadar to XSOAR).
```

Comment (tomneeman151293) [reply Packs/QRadar/ReleaseNotes/2_0_18.md:5]: Adding docs approved and doing your proposed suggestion

**Review by dantavori — APPROVED** [review]
